test/pensions/config/employmentHistory.unit.spec.jsx

```diff
@@ -0,0 +1,153 @@
+
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(16);
+  });
+
+  it('should not submit empty form', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    submitForm(form);
+
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(10);
+    expect(onSubmit.called).to.be.false;
+  });
+
+  it('should add another job', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_employer'), {
+      target: {
+        value: 'Smith'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_address_street'), {
+      target: {
+        value: '101 Elm st'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_address_city'), {
+      target: {
+        value: 'Northampton'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_address_state'), {
+      target: {
+        value: 'MA'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_address_postalCode'), {
+      target: {
+        value: '01060'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_jobTitle'), {
+      target: {
+        value: 'Professor'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_dateRange_fromMonth'), {
+      target: {
+        value: '1'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_dateRange_fromDay'), {
+      target: {
+        value: '1'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_dateRange_fromYear'), {
+      target: {
+        value: '2002'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_dateRange_toMonth'), {
+      target: {
+        value: '1'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_dateRange_toDay'), {
+      target: {
+        value: '1'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_dateRange_toYear'), {
+      target: {
+        value: '2003'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_daysMissed'), {
+      target: {
+        value: '3'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_annualEarnings'), {
```

Comment:
Yeah. I wonder how much effort we want to put into that, vs. just deciding to switch over to Enzyme.